amundsen_application/static/js/components/common/ResourceListItem/TableListItem/index.tsx

```diff
@@ -15,38 +13,41 @@ class TableListItem extends React.Component<TableListItemProps, {}> {
     super(props);
   }
 
-
-  /* TODO: We have to fix a bug with this feature. Commented out support.
-  const createLastUpdatedTimestamp = () => {
-    if (lastUpdated) {
-      const dateTokens = new Date(lastUpdated).toDateString().split(' ');
-      return (
-        <label>
-          {`${dateTokens[1].toUpperCase()} ${dateTokens[2]}`}
-        </label>
-      )
-    }
-    return null;
-  }*/
-
   getLink = () => {
-    const { item, logging } = this.props;
-    return `/table_detail/${item.cluster}/${item.database}/${item.schema_name}/${item.name}`
+    const { table, logging } = this.props;
+    return `/table_detail/${table.cluster}/${table.database}/${table.schema_name}/${table.name}`
       + `?index=${logging.index}&source=${logging.source}`;
   };
 
-
   render() {
-    const { item } = this.props;
+    const { table } = this.props;
+
+    // TODO - default last_updated timestamp is just for testing
+    table.last_updated = table.last_updated || Date.now();
+    const dateTokens = new Date(table.last_updated).toDateString().split(' ');
+    const lastUpdated = `${dateTokens[1]} ${dateTokens[2]}, ${dateTokens[3]}`;
+
     return (
-      <li className="list-group-item search-list-item">
-        <Link to={ this.getLink() }>
-          <img className="icon icon-color icon-database" />
-          <div className="resultInfo">
-            <span className="title truncated">{ `${item.schema_name}.${item.name} `}</span>
-            <span className="subtitle truncated">{ item.description }</span>
+      <li className="list-group-item">
+        <Link className="resource-list-item table-list-item" to={ this.getLink() }>
+          <img className="icon icon-database icon-color" />
+          <div className="content">
+            <div className="col-xs-12 col-sm-6">
+              <div className="main-title truncated">{ `${table.schema_name}.${table.name}`}</div>
+              <div className="description truncated">{ table.description }</div>
+            </div>
+            <div className="col-sm-3">
+              <div className="secondary-title">Frequent Users</div>
+              {/*TODO - Replace with a link to a real user*/}
+              <div className="description"><a href="#">Ash Ketchum</a></div>
+            </div>
+            <div className="col-sm-3">
+              <div className="secondary-title">Last Updated</div>
```

Comment:
This is supposed to be "Latest Data" according to the new design, which I hope was intentional and which I think is a better phrase. "Last Updated" might be misinterpreted to think this date was maybe the last time someone edited a.k.a. "updated" some metadata on that resource in Amundsen, when really this the date for which the table has the most recent data.